plugins/notifications-backend/src/service/router.ts

```diff
@@ -114,7 +136,7 @@ export async function createRouter(
       throw validation.errors;
     }
 
-    api.handleRequest(req as Request, req, res, next);
+    api.handleRequest(req as Request, req, res).catch(next);
```

Comment:
did u encounter a scenario where this change was needed? cause it is based on an example from the docs and it worked with all exceptions so far. if there's no real reason then please revert.

Reply:
With this change, we do not need to handle the `catch/next` multiple times inside the handler.
In other words, it makes less what-if flows inside of the handler, we can just chain promises there.

Before this patch, there was just a single promise per handler (IIRC) but with the authorization, we have +1.